### ISSUE TITLE
lazy claim should be unsigned extrinsic

### ISSUE THREAD

**Issue by brenzi** [encointer/pallets#101]
https://github.com/encointer/pallets/blob/976849b45ba5be04114f62c32c5f8b8fa864103f/ceremonies/src/lib.rs#L322

As all the meetups need to be claimed exactly once, it would be unfair to charge fees from the one that takes action to the benefit of all.
And execution only happens once, so no need to spam-prevent

**Comment by brenzi** [comment]
@clangenb please add to gesell-0.8.0 epic

**Comment by clangenb** [comment]
There are several ways we could go on about it. Even though we only execute the logic once, we still need to do spam-prevention; otherwise an attacker can fill entire blocks with invalid transactions and DOS the chain because there is no room for other transactions.

So the two options are:
* keep it signed and refund the fees if it was successful.
* perform all the validation logic in the signed extension, such that only valid extrinsics are included in the transaction pool.

This post nicely explains all the options: https://hackernoon.com/zero-cost-blockchain-transactions-are-a-reality-on-polkadot

tl;dr
I think implementing the signed extension logic is overly complicated. So I'd go for simply making the transaction zero-cost if the validation was successful.

**Comment by brenzi** [comment]
cash-back is fine as the call will come from someone who previously did have funds to register as a participant

**Comment by clangenb** [comment]
Cashback has been implemented a while ago: https://github.com/encointer/pallets/blob/7b59e07a12b82689472d519cf0bf71b9759f6c39/ceremonies/src/lib.rs#L544
